services/server/config/config.go

```diff
@@ -94,8 +95,9 @@ func (c *Config) GetVersion() int {
 
 // ValidateV2 validates the config for a v2 file
 func (c *Config) ValidateV2() error {
-	if c.GetVersion() != 2 {
-		return nil
+	version := c.GetVersion()
+	if version < 2 {
+		logrus.Warnf("deprecated version : `%d`, please switch to version `2`", version)
```

Comment:
Could you update deprecation table, then LGTM https://github.com/containerd/containerd/blob/master/RELEASES.md#deprecated-features

Reply:
Done!. thanks @AkihiroSuda